src/platforms/rcore_desktop_rgfw.c

```diff
@@ -643,7 +645,7 @@ Vector2 GetWindowScaleDPI(void)
 {
     RGFW_monitor monitor = RGFW_window_getMonitor(platform.window);
 
-    return (Vector2){((u32)monitor.scaleX)*platform.window->r.w, ((u32) monitor.scaleX)*platform.window->r.h};
+    return (Vector2){monitor.scaleX, monitor.scaleX};

```

Comment:
Not following raylib coding conventions, a space after `{` and before `}` should be used